docs/upgrade-notes/upgrade-rds-cluster-tf-postgres-13.md

```diff
@@ -0,0 +1,41 @@
+---
+id: upgrade-rds-cluster-tf-postgres-13
+title: Upgrade Database Cluster to PostgreSQL v13
+hide_title: false
+---
+
+Cumulus Core as of version > 18.1.0, and for the stable support release 16.1.x *only* for versions > v16.1.3 now supports and is tested against Aurora Postgres v13.   All users should update their datastores to this version as part of an upgrade process upon upgrading to release version 18.2.0 or 16.1.4.
```

Comment:
```suggestion
Cumulus Core as of version > 18.1.0** now supports and is tested against Aurora Postgres v13.   All users should update their datastores to this version as part of an upgrade process upon upgrading to release version 18.2.0.

**This update was also backported to the 16.1.x release chain starting with v16.1.3.
```

Reply:
Just trying to push to the latest version and make the v16 route a footnote.

Reply:
Hmm, fair enough.  👍🏻